addons/account/models/account_move.py

```diff
@@ -3574,6 +3575,7 @@ class AccountMoveLine(models.Model):
     tax_tag_invert = fields.Boolean(string="Invert Tags", compute='_compute_tax_tag_invert', store=True, readonly=False,
         help="Technical field. True if the balance of this move line needs to be "
              "inverted when computing its total for each tag (for sales invoices, for example).")
+    all_taxes_are_on_payment = fields.Boolean(compute='_compute_all_taxes_are_on_payment', store=True)
```

Comment:
I know that indexes on boolean might not be useful, but can't it be in this case?

Reply:
There is a chance that it would, but I don't think it worth to do it. Adding the field already should improve performance

Reply:
I tried, but it doesn't improve the speed